CMake/telesculptor-depends.cmake

```diff
@@ -17,3 +17,54 @@ include( kwiver-flags )
 include( kwiver-configcheck )
 include( telesculptor-utils ) # local utilities
 
+# We must find third party packages used by both KWIVER and TeleSculptor here
+# otherwise CMake will be unable to find those targets.
+# In the future, this should be exported by KWIVER.
+
+set(Qt5_MODULES Core Designer UiPlugin Widgets Svg Xml)
```

Comment:
note: the rest of this diff is purely relocated from `gui/CMakeLists.txt`.  There are no code changes to review.

Reply:
Where is this file used? I assume in the build itself, but is it involved in the `find_package` of this project?

Reply:
Are you asking if this is involved when someone tries to `find_package(TeleSculptor)`? (Note: I'm not sure that's even supported at this time; IIUC TeleSculptor isn't meant to be a library on its own.)

Reply:
Oh, I see. This is fixing it on the usage side, not the producer side :) . Carry on.

Reply:
Yes, my understanding is that getting KWIVER to properly export all of these targets conditioned on which parts of KWIVER you are using is a lot of work.  So we will fix it this way for now.